worker.py

```diff
@@ -1049,8 +1110,12 @@ def normalize_pokestop(raw):
     @staticmethod
     def check_captcha(responses):
         challenge_url = responses.get('CHECK_CHALLENGE', {}).get('challenge_url', ' ')
-        if challenge_url != ' ':
-            raise CaptchaException
+        success = responses.get('VERIFY_CHALLENGE', {}).get('success')
+        if challenge_url != ' ' and not success:
```

Comment:
yes the `challenge_url` is still in the request, even if the `VERIFY_CHALLENGE` was successfull